mods/ra/weapons/missiles.yaml

```diff
@@ -1,53 +1,13 @@
-Maverick:
-	ReloadDelay: 30
-	Range: 9c0
-	MinRange: 3c0
-	Report: missile7.aud
-	Burst: 2
-	BurstDelay: 7
-	ValidTargets: Ground, Water
-	Projectile: Missile
-		Speed: 256
-		Arm: 2
-		Blockable: false
-		ContrailLength: 10
-		Inaccuracy: 512
-		Image: DRAGON
-		HorizontalRateOfTurn: 5
-		CruiseAltitude: 2c0
-		RangeLimit: 14c410
-	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 70
-		ValidTargets: Ground, Water
-		Versus:
-			None: 30
-			Wood: 90
-			Light: 90
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
-	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion
-		ImpactSounds: kaboom25.aud
-		InvalidImpactTypes: Water
-	Warhead@4EffWater: CreateEffect
-		Explosions: med_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
-
-Dragon:
+^AntiAllMissile:
 	ReloadDelay: 50
 	Range: 5c0
 	MinRange: 0c512
 	Report: missile6.aud
-	ValidTargets: Ground, Water
+	ValidTargets: Ground, Water, Air
```

Comment:
I don't think we need to define this here, as it's overwritten by all weapons inheriting `^AntiAllMissile`.